API Blueprint Specification.md

```diff
@@ -478,6 +488,189 @@ Name and description of the API
+        ...
+        + `<key N>`: <value N>
+
+Where:
+
++ `<scheme name>` is the scheme name as written in [Authentication section](#def-authentication-section) under an [Action section](#def-action-section).
++ `<description>` is any **optional** Markdown-formatted description of the authentication scheme.
++ `<base scheme type>` is the authentication scheme type as expected by the API (e.g. "Basic Auth Scheme"). See below for all the available base authentication schemes.
++ `<key n>` represents a key of an customizable option in the base authentication scheme.
++ `<value n>` represents the value for the option with the key `<key n>`.
+
+Description of the base authentication schemes inherently defined in API Blueprint are defined below.
+
+#### Basic Authentication Scheme
+This is the [Basic Authentication Scheme](http://tools.ietf.org/html/rfc2617#section-2) as specified in [RFC 2617](http://tools.ietf.org/html/rfc2617).
+
+The customizable options for this scheme are:
+
++ `sample_username` represents a sample username value. "sample_user" is the **default**.
++ `sample_password` represents a sample password value. "sample_pass" is the **default**.
+
+##### Example
+
+    + basic (Basic Auth Scheme)
+        + sample_username: `bond`
+        + sample_password: `Iam007`
+
+#### Oauth2 Authentication Scheme
+This scheme uses the [Oauth2 Authorization Framework](http://tools.ietf.org/html/rfc6749) specified in [RFC 6749](http://tools.ietf.org/html/rfc6749) to get an authorization grant and uses it access the API.
+
+The customizable options for this scheme are:
+
++ `authorization_endpoint` is the authorization endpoint required for the authorization grant. "/authorize" is the **default**.
++ `access_token_endpoint` is the token endpoint used by client to obtain the access token. "/access_token" is the **default**.
++ `token_header_keyword` represents the string used instead of "Bearer" in "Authorization" header field according to [RFC 6750](http://tools.ietf.org/html/rfc6750#section-2.1). "Bearer" is the **default**.
++ `token_body_keyword` represents the parameter name used instead of "access_token" in the body of an API request according to [RFC 6750](http://tools.ietf.org/html/rfc6750#section-2.2). "access_token" is the **default**.
++ `token_query_keyword` represents the parameter name used instead of "access_token" in the query of an API request according to [RFC 6750](http://tools.ietf.org/html/rfc6750#section-2.3). "access_token" is the **default**.
++ `sample_token` represens a sample access token value. "sample_token" is the **default**.
+
+##### Example
+
+    + oauth (Oauth2 Auth Scheme)
+        + access_token_endpoint: `/token`
+        + token_header_keyword: `token`
+
+#### Bearer Authentication Scheme
+This scheme is popularly used when API consumers has access to their own unique access token which can be used as a bearer token as specified in [RFC 6750](https://tools.ietf.org/html/rfc6750#section-2.1).
```

Comment:
> This scheme is popularly used when API consumers has access to their own unique access token which can ...

There is no room for justification in the spec. No need to justify ourselves. Throw this part out, keep it simple.